l10n-pocket/en/pocket/privacy.ftl

```diff
@@ -0,0 +1,89 @@
+pocket-privacy-use-of-cookies = Use of Cookies and Analytics
+pocket-privacy-use-of-information = Use of Information
+pocket-privacy-sharing = Sharing of Information
+pocket-privacy-keeping = Keeping of Information
+pocket-privacy-transfer = Transfer of Information
+pocket-privacy-security = Security
+pocket-privacy-phishing = Phishing
+pocket-privacy-managing-my-privacy = Managing My Privacy and Email Settings
+pocket-privacy-amendments = Amendments to Policy
+pocket-privacy-contact = Contact
+
+pocket-privacy-you-are-not-required = You are not required to create a user account or provide any personal information in order to visit the { -brand-name-pocket } website. You are required to create a user account, however, to use our application. During registration, you are required to provide a name, password and valid email address.
+pocket-privacy-if-you-choose = If you choose to share any articles, videos, or other content with another person, we will ask you to provide a valid email address for that person. We may also ask you to provide your username and password for third party websites in order to access the articles and information published on such sites. We also collect information if you contact us for support or help, submit an application or otherwise communicate with us. The information which you supply to us enables us to customize and enhance the services we provide to you. In addition, it helps us communicate with you about our news, events, promotions and new products.
+pocket-privacy-if-you-choose-to-subscribe = If you choose to subscribe to our paid subscription service, our third party payment processors will collect and handle your credit card and related payment information. Such third party processors’ use of your personal information is governed by their respective privacy policies.
+pocket-privacy-in-addition = In addition to the information that you provide to us when you register for a user account, we collect information about the URLs, titles and content of the web pages and other information you save to { -brand-name-pocket }. The types of information we collect includes your browser type, device type, time zone, language, and other information related to the manner in which you access the { -brand-name-pocket-technologies }. If you are on a mobile device, we collect the advertising identifiers provided by Apple on iOS and by Google on Android. You can change this identifier in your device settings. We also collect information about your use of the { -brand-name-pocket-technologies } so that we can provide our services. For example, as a part of providing { -brand-name-pocket }’s syncing features, we sync information about the items that you save and view within { -brand-name-pocket } so that your list, tags, scroll position, and other account and usage information may be synced across all of your devices.
```

Comment:
Issue (blocking): brand names need adding here for `Apple on iOS and by Google on Android`